cardano-cli/test/cli/genesis-key-gen-delegate/run

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh -u
+
+cwd=$(dirname "$0")
+
+# shellcheck source=/dev/null
+. "${cwd}/../core/common"
+
+# shellcheck disable=SC2154
+banner "${testname}"
+
+error=0
+
+# Does it make sense to add this test ? After checking it looks like a clone of:
+# https://github.com/input-output-hk/cardano-node/blob/master/cardano-cli/test/cli/node-key-gen/run
+# the only difference is the title. which I added to checks at the bottom - is it worth to add this then ?
+
+${CARDANO_CLI} shelley genesis key-gen-delegate \
+    --verification-key-file "${TEST_DIR}/key-gen.vkey" \
+    --signing-key-file "${TEST_DIR}/key-gen.skey" \
+    --operational-certificate-issue-counter "${TEST_DIR}/op-cert.counter"
+
+assert_file_exists "${TEST_DIR}/key-gen.vkey"
+assert_file_exists "${TEST_DIR}/key-gen.skey"
+assert_file_exists "${TEST_DIR}/op-cert.counter"
+
+# Check type:
+assert_string 'Node operator verification key' "${TEST_DIR}/key-gen.vkey"
+assert_string 'Node operator signing key' "${TEST_DIR}/key-gen.skey"
+assert_string 'Node operational certificate issue counter' "${TEST_DIR}/op-cert.counter"
+
+# Check title:
```

Comment:
This one (and probably some below too) looks like clone of this test `node-key-gen/run` - so not sure if it is needed - **however** - the titles for those files are different, so we can check for those differences if we want to

Reply:
Worth removing that comment and keeping the test :D.

Tests are there to make sure we know when we break something. We should keep this.

Reply:
The title is free form test. The user can edit and modify it. We should not be checking it.

Reply:
I have pushed a fix that removes that title check. I will merge this once CI is happy.